### ISSUE TITLE
No valid engine configs for Norm_Fwd_

### ISSUE THREAD

**Issue by EnricoDeg** [NVIDIA/cudnn-frontend#133]
Hello,

I tried to test batch norm for fwd training using NCHW=160,35,48,24 with half and I get the following runtime error:

cudnn._compiled_module.cudnnGraphNotSupportedError: No valid engine configs for Norm_Fwd_
{"engineId":0,"smVersion":890,"knobChoices":{"CUDNN_KNOB_TYPE_OCCUPANCY":-1,"CUDNN_KNOB_TYPE_ARRAY_SIZE_PER_THREAD":-1,"CUDNN_KNOB_TYPE_NUM_C_PER_BLOCK":-1}}

when I try to create an execution plan for the graph. The problem seems to be related to the C dimension value.
Is this a problem with the cuDNN frontend or does it come from the backend?
Do you have a suggestion on how to run this case?

**Comment by GeezYuven** [comment]
Have you solved this problem? What was the reason?
I encountered this problem when using batchnorm for inference.

**Comment by GeezYuven** [comment]
Is the single-operator batchnorm_inference currently not supported by the Graph API? and after enabling the backend log, the log also seems to indicate INVALID_OPERATION_MODE

**Comment by Anerudhan** [comment]
Thanks folks. Closing as stale and answered. Please re-open if needed
